client/src/Header.js

```diff
@@ -44,6 +59,17 @@ function Header() {
             About
           </Link>
         </li>
+        {loading ? (
+          <></>
+        ) : (
+          categories.map((category) => (
+            <li className={classNames('_item', { active })}>
+              <Link onClick={() => setActive(false)} to={`/categories/${category.slug}`}>
+                ${category.name}
```

Comment:
This `$` is showing up in the output... was that intentional? Otherwise, let's remove it, no...?